### PR TITLE
fixed issue #21558 - Navigation issue of review from product listing …

### PR DESCRIPTION
**Pull request by ghost**
### Description (*)
I have sent second parameter as true if the url has hash "#reviews" to make it scroll to proper position.

### Fixed Issues (if relevant)
1. magento/magento2#21558: Navigation issue of review from product listing when click on review count

### Manual testing scenarios (*)
1. Navigate to Product listing page
2. Click on review count link
3. It will redirect to product page and scroll to reviews.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Review/view/frontend/web/js/process-reviews.js [hunk 1]
```diff
@@ -41,7 +41,11 @@ define([
             requiredReviewTabRole = 'tab';
 
         if (reviewTab.attr('role') === requiredReviewTabRole && reviewTab.hasClass('active')) {
-            processReviews(config.productReviewUrl);
+            if (location.hash === '#reviews') {
+                processReviews(config.productReviewUrl, true);
+            } else {
+                processReviews(config.productReviewUrl);
```
Comment (orlangur) [inline app/code/Magento/Review/view/frontend/web/js/process-reviews.js]: Can we replace full block with 

```
processReviews(config.productReviewUrl, location.hash === '#reviews');
```

?

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Review/view/frontend/web/js/process-reviews.js (hunk 1 above)
Comment (ghost) [reply app/code/Magento/Review/view/frontend/web/js/process-reviews.js]: @orlangur changes applied in https://github.com/magento/magento2/commit/79d323949248c80719def6d3c5c69033ad731ba4

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-5082](https://jira.corp.magento.com/browse/ENGCOM-5082) has been created to process this Pull Request

**Review by orlangur — CHANGES_REQUESTED** [review]
Please squash changes into a single commit so that we have perfectly clean history 😉

**Comment by ghost** [comment]
@orlangur  I have created another PR with the mentioned changes because I was having issue while squashing commits 😬
https://github.com/magento/magento2/pull/22794

**Pull request closed without merging**